tests/test_signing.py

```diff
@@ -0,0 +1,97 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
+
+
+import tempfile
+from collections.abc import Mapping
+from pathlib import Path
+
+import pytest
+
+from mkosi.run import find_binary, run
+
+from . import Image, ImageConfig
+
+pytestmark = pytest.mark.integration
+
+
+def test_signing_checksums_with_sop(config: ImageConfig) -> None:
+    assert config.tools is not None, "Needs config.tools to check for SOP implementation"
```

Comment:
:thinking: I've read #3109 and maybe I'm holding something wrong but this assertion keeps failing (locally). Do I need to enable something for the test to get the `mkosi.tools`?

Reply:
You just need to pass `root=config.tools` to `find_binary()` that's it, no need for assetions. It might or might not be set depending on whether mkosi.tools exists or not.